plaidml/op/op_test.cc

```diff
@@ -222,6 +222,43 @@ ConvolutionParams convParams[] = {
           conv.name("res2a_branch1").strides({1, 1}).autopad_mode(op::AutoPadMode::VALID);
         },
     },
+    // FIXME: this one takes a very long time to run.
+    // {
+    //     // 3D convolution
+    //     DType::FLOAT32,
+    //     {1, 32, 224, 224, 3},
+    //     {7, 7, 7, 3, 64},
+    //     [](op::convolution& conv) {
+    //       conv.strides({2, 2, 2}).autopad_mode(op::AutoPadMode::EXPLICIT).manual_padding({2, 3, 2, 3, 2, 3});
+    //     },
+    // },
+    {
+        // 3D convolution
+        DType::FLOAT32,
+        {1, 3, 1, 1, 1024},
+        {1, 1, 1, 1024, 400},
+        [](op::convolution& conv) {
+          conv.strides({1, 1, 1}).autopad_mode(op::AutoPadMode::EXPLICIT).manual_padding({0, 0, 0, 0, 0, 0});
+        },
+    },
+    {
+        // 3D convolution
+        DType::FLOAT32,
+        {1, 16, 56, 56, 64},
+        {1, 1, 1, 64, 64},
+        [](op::convolution& conv) {
+          conv.strides({1, 1, 1}).autopad_mode(op::AutoPadMode::EXPLICIT).manual_padding({0, 0, 0, 0, 0, 0});
+        },
+    },
+    {
+        // 3D convolution
+        DType::FLOAT32,
+        {1, 16, 56, 56, 64},
+        {3, 3, 3, 64, 192},
+        [](op::convolution& conv) {
+          conv.strides({1, 1, 1}).autopad_mode(op::AutoPadMode::EXPLICIT).manual_padding({1, 1, 1, 1, 1, 1});
+        },
+    },
```

Comment:
I'm still seeing this test segfaulting in the PR build. Do we want to add this one as a todo?

Reply:
I'm trying to repro this locally, it seems to pass fine for me, I'll dig into this one a bit.